### ISSUE TITLE
Parser error when using "as <namespace>" with @use in .scss files

### ISSUE THREAD

**Issue by gustavoferrara** [Sysmagine/SemanticDiff#31]
**Describe the Bug**
Parser reporting an error with the `as` keyword when trying to choose a namespace with `@use`.

**To Reproduce**
1. Open the diff on a .scss file which contains `@use "./file" as namespace`.

**Expected Behavior**
Should parse correctly.

**Actual Behavior**
Returns a parsing error and highlights the `as` keyword.

**SemanticDiff Version**
0.8.4

**VS Code Information**
```
Version: 1.80.2 (user setup)
Commit: 2ccd690cbff1569e4a83d7c43d45101f817401dc
Date: 2023-07-27T20:40:28.909Z
Electron: 22.3.14
ElectronBuildId: 22695494
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.19045
```

**Comment by mmueller2012** [comment]
Thank you for reporting this bug. The scss parser didn't support namespaces in the `@use` statement yet. We have already merged a fix and it should be available in the next release.

Was this the only parser error you encountered? The namespace support is still a bit incomplete, so I just want to make sure you don't run into the next problem when the fix is available.

**Comment by gustavoferrara** [comment]
Regarding to the scss parser no other issues so far. Separately on .tsx files, if you add a parent div which causes indentation changes inside the jsx, the diff will detect the added spaces as changes. It isn't something that major but would be nice to have it filtered out especially on bigger files.

![image](https://github.com/Sysmagine/SemanticDiff/assets/73325370/bd24b5dd-aef4-49b5-acda-5bb18befbf52)

**Comment by mmueller2012** [comment]
We have just released SemanticDiff 0.8.5 which adds support for namespaces in `@use` statements. Can you confirm that it works for you?

**Comment by gustavoferrara** [comment]
Yeah it's parsing correctly now, thank you!
